charts/trusted-artifact-signer/ci/ci-values.yaml

```diff
@@ -36,10 +38,6 @@ scaffold:
             ClientID: sigstore
             Type: email
   rekor:
-    clusterMonitoring:
-      enabled: false
-      http:
-        enabled: false
     server:
```

Comment:
```
    clusterMonitoring:
      enabled: false
      http:
        enabled: false
```

I noticed that the http values are removed here but not in examples/values-kind-sigstore.yaml, is this right ?

Reply:
I think those were placed in error from the beginning : ) so I'll remove that, also, thanks

Reply:
removed!